content/2023/contents/venue.mdx

```diff
@@ -1,7 +1,76 @@
+import Image from 'next/image';
+import venueImg from '@/assets/venue.png';
+import confImg1 from '@/assets/conference_1.png';
+import confImg2 from '@/assets/conference_2.png';
+
 # Venue
 
-WIP
+<div className="tablet:tw-mb-5">
+    <div div className="tablet:tw-flex tablet:tw-items-center tablet:tw-justify-center">
+    [MIRC](https://eic2.web.nycu.edu.tw/) of National Yang Ming Chiao Tung University 
+    </div>
+
+    <div div className="tablet:tw-flex tablet:tw-items-center tablet:tw-justify-center">
+    No. 1001, Ta Hsueh Road, Hsinchu, Taiwan
+    </div>
+</div>
+
+# Transportation
+
+<div div className="tablet:tw-flex tablet:tw-items-center tablet:tw-justify-center">
+  <Image
+    className="tablet:tw-mr-5 tablet:tw-w-4/5 tablet:tw-mt-14"
+    src={venueImg}
+    alt="venue"
+  />
+</div>
+
+## By Car
+
+Navigate to [MIRC](https://eic2.web.nycu.edu.tw/)：[Google Map](https://maps.app.goo.gl/MttEZQraQ3usSbj47?g_st=ic)  
+
+**School Parking Guidelines**   
+All white parking spaces (with white lines) within the campus are available for parking. Only vehicles with valid parking permits for faculty and staff issued by the school in the current year are allowed to park in the blue parking spaces (with blue lines). Parking is prohibited on red lines, hatched yellow lines, and pedestrian walkways.  
+
+## By THSR
+
+After arriving at THSR Hsinchu station, there are two ways to get to NYCU Guangfu Campus.
+
+1. Take Bus No. 182 or the bus BL1 to the "Guo Gou" stop. Get off and walk to [MIRC](https://eic2.web.nycu.edu.tw/). Click to see [the reference route]((https://maps.app.goo.gl/GU7nbFYjc5bPLJPD8)).   
```

Comment:
The link of reference route here is incorrect (this is for train) and invalid.

Reply:
Ok. I've corrected it on the latest commit.